project_multifamily_beta/housing_characteristics/Insulation Wall.tsv

```diff
@@ -1,91 +1,181 @@
+CR07	2000s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.390000	0.030000	0.580000	0.000000
+CR08	2000s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.600000	0.220000	0.180000	0.000000
+CR09	2000s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.670000	0.150000	0.170000	0.000000
+CR10	2000s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.410000	0.200000	0.390000	0.000000
+CR11	2000s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.470000	0.020000	0.510000	0.000000
+CR02	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.310000	0.170000	0.310000	0.140000	0.070000	0.000000
+CR05	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR03	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR04	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR06	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR07	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR08	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR09	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR10	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR11	1940s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000	0.000000	0.000000	0.000000	0.000000
+CR02	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	1.000000	0.000000
+CR05	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.140000	0.140000	0.710000	0.000000
+CR03	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.200000	0.050000	0.750000	0.000000
+CR04	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.370000	0.320000	0.310000	0.000000
+CR06	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.090000	0.000000	0.910000	0.000000
+CR07	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.390000	0.030000	0.580000	0.000000
+CR08	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.600000	0.220000	0.180000	0.000000
+CR09	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.670000	0.150000	0.170000	0.000000
+CR10	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.410000	0.200000	0.390000	0.000000
+CR11	2010s	Masonry	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.470000	0.020000	0.510000	0.000000
```

Comment:
Add "Created manually using data from Home Innovation Research Labs Builder Practices Reports (1990s and 2000s) and NAHB Builder Practices Reports (1980s). Expert elicitation used for all other years." or similar

Reply:
How soon are you wanting to do a run on masonry walls? I can add the tag, but I'm not sure which format to use or when Tony will have the update made (the difference is that he wants the tag as a comment with a hashtag). 

As a side note, I really dislike having the tag directly in the TSV. It's just a pain to do calculations with the TSV when you have to strip out the tag first. I'd rather have a central reference file that listed all the sources and automatically populated updates when tsv_maker.py was run.